tests/examples_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2021 NMPolicy Authors.
```

Comment:
2022 (unless the commit was authored in 2021 already).

Reply:
I will change all the headers of this PR.